src/main/java/org/prebid/server/auction/IpAddressHelper.java

```diff
@@ -87,9 +87,8 @@ public String maskIpv4(String ip) {
                 return ip;
             }
         }
-        return String.format("%s%s", maskedIp,
-                IntStream.range(0, 1).mapToObj(ignored -> "0")
-                        .collect(Collectors.joining(".", ".", "")));
+        return maskedIp + IntStream.range(0, 1).mapToObj(ignored -> "0")
+                .collect(Collectors.joining(".", ".", ""));
```

Comment:
Looks like `IntStream.range(0, 1).mapToObj(ignored -> "0").collect(Collectors.joining(".", ".", ""))` can be converted to `".0"`